cmake/projects/Protobuf/hunter.cmake

```diff
@@ -7,29 +7,51 @@ include(hunter_download)
 include(hunter_pick_scheme)
 
 hunter_add_version(
-  PACKAGE_NAME
-  Protobuf
-  VERSION
-  "3.0.0-beta-2"
-  URL
-  "https://github.com/hunter-packages/protobuf/archive/v3.0.0-beta-2-hunter-1.tar.gz"
-  SHA1
-  15752454ef0cf119b58136e2f4ffe97baeaf07cd
-  )
+    PACKAGE_NAME
+    Protobuf
+    VERSION
+    "3.0.0-p1"
+    URL
+    "https://github.com/hunter-packages/protobuf/archive/v3.0.0-hunter-3.tar.gz"
+    SHA1
+    e3d5ab642e0295d16b86a4a55e06c3ee026ff249
+)
 
 hunter_add_version(
-  PACKAGE_NAME
-  Protobuf
-  VERSION
-  "3.0.0-beta-2-p1"
-  URL
-  "https://github.com/hunter-packages/protobuf/archive/v3.0.0-beta-2-hunter-2.tar.gz"
-  SHA1
-  dbae4995254f351918d0462b308b561dbab71b3f
-  )
+    PACKAGE_NAME
+    Protobuf
+    VERSION
+    "3.0.0"
+    URL
+    "https://github.com/hunter-packages/protobuf/archive/v3.0.0-hunter-2.tar.gz"
+    SHA1
+    6fb368b171aa10f612d777f079d6896f37b85c55
+)
+
+hunter_add_version(
+    PACKAGE_NAME
+    Protobuf
+    VERSION
+    "3.0.0-beta-2"
+    URL
+    "https://github.com/hunter-packages/protobuf/archive/v3.0.0-beta-2-hunter-1.tar.gz"
+    SHA1
+    15752454ef0cf119b58136e2f4ffe97baeaf07cd
+)
+
+hunter_add_version(
+    PACKAGE_NAME
+    Protobuf
+    VERSION
+    "3.0.0-beta-2-p1"
+    URL
+    "https://github.com/hunter-packages/protobuf/archive/v3.0.0-beta-2-hunter-2.tar.gz"
+    SHA1
+    dbae4995254f351918d0462b308b561dbab71b3f
+)
 
 
-hunter_cmake_args(Protobuf CMAKE_ARGS  CMAKE_ARGS protobuf_BUILD_TESTS=OFF)
+hunter_cmake_args(Protobuf CMAKE_ARGS CMAKE_ARGS protobuf_BUILD_TESTS=OFF)
```

Comment:
`hunter_cmake_args(Protobuf CMAKE_ARGS CMAKE_ARGS protobuf_BUILD_TESTS=OFF)` should be just `hunter_cmake_args(Protobuf CMAKE_ARGS protobuf_BUILD_TESTS=OFF)`, I guess I missed that on previous pull request.